lib/server/ServerAjax.js

```diff
@@ -39,7 +39,8 @@ var ServerAjax = {
             requestOptions.url = url;
 
             return Testable.requestPromise(requestOptions)
-                .then(function(incomingMessage) {
+                .then(function(responseArgs) {
```

Comment:
spread would work too. to be honest, I'm planning to move to a different promise library in https://github.com/bloomberg/brisket/pull/285 so it's better to use standard methods i think